catalog/tests/test_models.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+from catalog.models import Author
+
+
+class AuthorModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        # Set up non-modified objects used by all test methods
+        Author.objects.create(first_name="Big", last_name="Bob")
+
+    def test_first_name_label(self):
+        author = Author.objects.get(id=1)
+        field_label = author._meta.get_field("first_name").verbose_name
+        self.assertEqual(field_label, "first name")
```

Comment:
EM tìm hiểu thêm 1 số các assertion methods khác mà class TestCasse cung cấp

Reply:
- assertTrue: Kiểm tra đk đúng
- assertFalse: Kiểm tra đk sai
- assertNotEqual(a,b): kiểm tra a!=b
- assertIn(a,b): kiểm tra a in b
- assertNotIn(a,b)
- assertIsNone(a): kiểm tra a là None
- asserIsNotNone(a)